src/components/pages/DataVisualizations/YearLimitsSlider/index.jsx

```diff
@@ -0,0 +1,3 @@
+import MainBar from './MainBar';
+
+export { MainBar as default };
```

Comment:
Just want to make sure every file deleted was intended to be deleted and we did not just want to maintain the version on main. Deleting a file is different than not including it in your PR. Just want to make sure, as these will be deleted completely from `main`.

Reply:
Thank you for checking twice!